erpnext/patches/v12_0/add_einvoice_summary_report_permissions.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
```

Comment:
remove `future` and `six` imports. It's right now working only because some package has those dependency, we don't. It will start breaking randomly in future.

Reply:
Done